### DISCUSSION TITLE
How to automatically read the title in <directory>/index.md

### DISCUSSION THREAD

**Discussion by ayangweb** [web-infra-dev/rspress#1498] [Q&A]
仓库地址：https://github.com/EcoPasteHub/EcoPaste-Docs/blob/master/docs/problem/macos/_meta.json

大佬们好，因为我的一个 md 里有很多的图片，所以我就单独放在一个目录里方便管理，但是这样的话我目前测试的侧边导航想要出来文字，必须手动指定 label，没办法自动识别 [`docs/problem/macos/damage/index.mdx`](https://github.com/EcoPasteHub/EcoPaste-Docs/blob/master/docs/problem/macos/damage/index.mdx) 里的 title 了，这个有什么简单的解决方案实现自动识别吗？

<img width="1541" alt="iShot_2024-10-18_10 04 48" src="https://github.com/user-attachments/assets/68a467d2-548d-4e76-b484-ad5bec1be4c3">

<img width="1541" alt="iShot_2024-10-18_10 11 23" src="https://github.com/user-attachments/assets/45c56162-5b35-406d-93ea-d7af63094b56">

**Comment by Timeless0911** [comment]
你把它作为 file ，实际放在文件夹里，是在利用 `foo/` 自动跳转到 `foo/index` 的行为，那么就需要你手动写 `label`，你也可以作为文件夹去展开，也就是

```
{
    "type": "dir",
    "name": "damage",
    "label": "damage"
  }
```

![image](https://github.com/user-attachments/assets/5e59070d-cdb2-419f-bc9b-585f9e999655)

**Comment by ayangweb** [comment]
谢谢，已解决，我把图片放到 public 了，和 #1500 一起解决！
